pkg/protocol/group/group.go

```diff
@@ -1,5 +1,19 @@
+// Package group contains an implementation of a generic protocol group
```

Comment:
This is true that the `pkg/protocol/group` is used by DKG at the moment. However, I think it's wrong if we couple it with DKG this way since it closes possible reuse of that code in the future. This package has a generic code that can be used by any group performing an interactive protocol that needs recording misbehaving members. Note that the `pkg/protocol/state` depends on it as well and if we name it `pkg/protocol/dkg` we'll also tie the state machine code with DKG.

Reply:
We can revisit it when we'll be working on ECDSA signing. If we do not reuse any of these components for signing, I would rename the package to `dkg`. From what I see, the state machine needs only `group.MemberIndex` and I think `MemberIndex` should stay in `group`. All the inactivity/disqualification marking and message filtering are currently specific to DKG and that's why I was thinking about renaming the package and making it explicit what they are used for.